### PR TITLE
add videochat and pdf to poster 1759

### PR DESCRIPTION
**Pull request by rsetton**

### CODE REVIEW

**Comment by effigies** [comment]
Are all of these changes from the original, just the PDF, or some subset?

**Comment by rsetton** [comment]
just the pdf!

File: posters-overrides.json [hunk 1]
```diff
@@ -5003,7 +5003,30 @@
...
+    "Tian Ge",
+    "Richard Leahy",
+    "Gary Turner",
+    "Nathan Spreng"
+   ],
+   "keywords": [
+    "cognition",
+    "functional mri"
+   ],
+   "software-demo": ""
```
Comment (effigies) [inline posters-overrides.json]:
```suggestion
   "number": 1759,
   "pdf": "https://github.com/rsetton/misc/raw/master/OHBM_L%26R.pdf"
```

**Comment by effigies** [comment]
Looks like you've disabled edits from maintainers, so I can't accept that suggestion. Could you give it a shot?

**Comment by effigies** [comment]
Thanks!

**Comment by rsetton** [comment]
Thank you!

**Pull request merged**